diff --git a/dstar/SeBa.C b/dstar/SeBa.C
index c6bd16e..7b9d8ff 100644
--- a/dstar/SeBa.C
+++ b/dstar/SeBa.C
@@ -172,7 +172,7 @@ local bool  evolve_binary(dyn * bi,
       time = Starlab::min(time+dt, end_time);
 
       ds->evolve_element(time);
-
+      ds->dump(SeBa_outfile, true);
       if (stop_at_merger_or_disruption &&
 	  (ds->get_bin_type() == Merged || 
 	   ds->get_bin_type() == Disrupted))
diff --git a/sstar/starclass/constants.C b/sstar/starclass/constants.C
index d81c8a3..e317a13 100644
--- a/sstar/starclass/constants.C
+++ b/sstar/starclass/constants.C
@@ -494,7 +494,7 @@ real stellar_evolution_constants::safety(safety_parameter ps) {
           break;
     case minimum_inversion_precision:        return 0.01;// 1%
           break;
-    case number_of_steps:                   return 10;// error on mass smaller than a few percent
+    case number_of_steps:                   return 200;// error on mass smaller than a few percent
           // for M > 40 need n>=20 to get an error < ~7%
           // possibly other values for non_solar metallicities
         break;
